auto-diagram/src/App.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+
+  <div id="app">
+
+      <div class="content-box-appdev">
+          <div class="box-heading"><h2>Application Development</h2></div>
+
+          <div class="content-box-collaboration">
+              <div class="collaboration-wrapper">
+                  <div class="sub-box-heading"> <h2>COLLABORATION</h2> </div>
+                  <app-collaboration
+                    v-for="item in input_data.stack.collaborationTools"
+                    v-bind:struct="item">
+                  </app-collaboration>
+              </div>
+          </div>
+          <div class="content-box-environment">
+             <div class="sub-box-heading"> <h2>ENVIRONMENT</h2> </div>
+                 <app-environment :applications="input_data.stack.appContainers"
+                    v-for="item in input_data.stack.environments"
+                    v-bind:struct="item">
+                  </app-environment>
+              </div>
+          <div class="content-box-source">
+              <div class="source-wrapper">
+                  <div class="sub-box-heading"> <h2>SOURCE CONTROL</h2> </div>
+                  <app-source
+                    v-for="item in input_data.stack.sourceControl"
+                    v-bind:struct="item">
+                  </app-source>
+              </div>
+          </div>
+
+          <div class="content-box-test">
+      	<div class="sub-box-heading"><h2>PIPELINE</h2></div>
+              <div class="test-wrapper">
+                  <app-test
+                    v-for="item in input_data.stack.testLibraries"
```

Comment:
This and other for loops trigger a warning:

> Module Warning (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
(Emitted value instead of an instance of Error) <app-test v-for="item in input_data.stack.testLibraries">: component lists rendered with v-for should have explicit keys. See https://vuejs.org/guide/list.html#key for more info.

This doesn't have to be fixed in this PR but should be addressed eventually.